packages/breadcrumbs/src/styled/StyledBreadcrumbItem.ts

```diff
@@ -6,23 +6,34 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
+import { stripUnit } from 'polished';
+import { getColor } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.item';
 
 export interface IStyledBreadcrumbItemProps {
   isCurrent?: boolean;
 }
 
-export const StyledBreadcrumbItem = styled.li.attrs<IStyledBreadcrumbItemProps>(props => ({
+export const StyledBreadcrumbItem = styled.li.attrs(() => ({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb__item'], {
-    // State
-    [BreadcrumbStyles['is-current']]: props.isCurrent
-  })
+  'data-garden-version': PACKAGE_VERSION
 }))<IStyledBreadcrumbItemProps>`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+  ${props => `
+    line-height: ${(props.theme.space.base * 5) / stripUnit(props.theme.fontSizes.md)}}
+    color: ${props.isCurrent ? getColor(props.theme.colors.neutralHue, 600) : 'inherit'}
+    & a {
+      &:any-link {
```

Comment:
I'm assuming this came from the CSS side of things, but doesn't this break in IE11 and Edge https://developer.mozilla.org/en-US/docs/Web/CSS/:any-link ?

Reply:
It would be good if you break this out to an `anyLinkStyles` function – similar to what we did with postCSS [input range pseudo](https://github.com/jonathantneal/postcss-input-range) here: https://github.com/zendeskgarden/react-components/blob/1067eae528c03996c34c364046bdbff8c756e4c0/packages/forms/src/styled/range/StyledRangeInput.js#L34-L48

Reply:
@jzempel thanks for the suggestion! I added an `anyLinkStyles` function that takes a string template of CSS and applies it to the `any-link` pseudo-class selectors. What do you think about the following?

```css
/**
 * 1. CSS targets Microsoft Edge
 */
const anyLinkStyles = (styles: string) => `
  & a {
    &:any-link { ${styles} }
    &:any-link:hover { ${styles} }
    &:any-link:focus { ${styles} }
    &:link { ${styles} } /* [1] */
    &:visited { ${styles} } /* [1] */
    &:focus:link { ${styles} } /* [1] */
    &:hover:link { ${styles} } /* [1] */
    &:focus:visited { ${styles} } /* [1] */
    &:hover:visited { ${styles} } /* [1] */
  }
`;
```